rust-runtime/aws-smithy-types/src/lib.rs

```diff
@@ -13,455 +13,18 @@
     unreachable_pub
 )]
 
-use crate::error::{TryFromNumberError, TryFromNumberErrorKind};
-
-mod blob;
-pub use blob::Blob;
-
 pub mod base64;
+mod blob;
 pub mod date_time;
+pub use date_time::*;
```

Comment:
What is this change for?

Reply:
It causes a import error without it.

If you check this file(smithy-rs/rust-runtime/aws-smithy-types/src/date_time/format.rs), you can find 5 of `use crate::DateTime`.

mod.rs file has it one too.

I'm not quite sure how it ended up this way though.